WordPress/src/main/res/values/strings.xml

```diff
@@ -2561,7 +2561,7 @@
     <string name="domain_contact_information_postal_code_hint">Postal code</string>
     <string name="domain_contact_information_register_domain_button">Register domain</string>
     <string name="domain_registration_result_title">Congratulations on your purchase!</string>
-    <string name="domain_registration_result_description">Your new domain &lt;b&gt;%s&lt;/b&gt; is being setup. It may take up to 30 minutes for your domain to start working.</string>
+    <string name="domain_registration_result_description">Your new domain &lt;b&gt;%s&lt;/b&gt; is being set up. It may take up to 30 minutes for your domain to start working.</string>
```

Comment:
Note that this change in the copy won't have a change to be re-translated until next code freeze (for `20.0`), since the English copies have already been frozen and sent to GlotPress for translation at the start of this sprint (and it's way too late in the sprint to get this retranslated before Friday).

That is probably not a big deal given what this change is, and the fact that it's a typo fix in the English language, so the translations that have probably already been submitted in GlotPress by the polyglots/translators for this string probably already translated this correctly (using the a passive verb form of their language rather than the noun) in the other locales. Except maybe in `en-GB`, `en-AU` and similar English variants, where many copies are similar to the original `en-US` ones and the `en-GB`/`en-AU` translators might have missed that typo and just copied the original string verbatim?

Anyway, just wanted to still give a heads up there to remind about the impact of the translation pipeline and its relation to the release cycle to let you know.

PS: That change will also mean that during next code freeze, that string will be marked as "fuzzy" in GlotPress (since the original would have only be changed very slightly, so it won't _completely_ invalidate all the translations for it but instead just mark all of them as needing to be _revalidated_ by polyglots in each locale so that they can double-check their past translation for `setup` still holds with the new `set up`). Usually polyglots are quick at re-approving a translation that was previously translated and marked fuzzy, so that shouldn't impact the `20.0` much when that will happen after next code freeze… but again, I thought it'd be interesting to share here how that all works 🙂

Reply:
Thank you @AliSoftware for detailed explanation 🙇   It is fine as long as it is done in the next release as well.

Reply:
Actually I've just realized by now that my explanation above was not completely correct, because nowadays we update the frozen strings when we do a new beta (like the `rc-4` I just did) exactly to handle cases like this where strings are updated in the `release/*` branch 🙃

My explanation and notes above still hold though, except that the translations turning into "fuzzy" state in GlotPress will now happen soon after the release branch is merged back into `trunk` after a new beta (which happened a couple hours ago for `rc-4` already) — rather than waiting for the next code freeze for this to happen — giving polyglots one last chance before the final release build to translate any new string, or in this case revisit/re-validate that their already-previously-translated copy is still valid even after the change.

Which has its pros and cons, because when that happens early in the sprint they usually have enough time to update their translations … but when it happens late in the sprint the strings that gets invalidated/fuzzied might not be re-evaluated/validated in time for the final build (which, if that happens, will lead to the app not including the fuzzied translations and thus showing the English copy for that string in all locales even if it has a proper translation before that typo fix…)